### PR TITLE
Fix docs typos

### PR DESCRIPTION
**Pull request by szepeviktor**
@powerivq From #39602

See `typos` config in that PR!

### CODE REVIEW

**Review by powerivq — APPROVED** [review]

File: docs/spec/amp-email-actions-and-events.md [hunk 1]
```diff
@@ -431,7 +431,7 @@ event.response</pre></td>
   </tr>
   <tr>
     <td><code>changeToLayoutContainer</code></td>
-    <td>Update's <code>amp-list</code>'s layout to <code>layout="CONTAINTER"</code> to allow <a href="../extensions/amp-list/amp-list.md#dynamic-resizing">dynamic resizing</a>.</td>
+    <td>Update's <code>amp-list</code>'s layout to <code>layout="CONTAINER"</code> to allow <a href="../extensions/amp-list/amp-list.md#dynamic-resizing">dynamic resizing</a>.</td>
```
Comment (powerivq) [inline docs/spec/amp-email-actions-and-events.md:434]: The link should be `../../extensions/amp-list/amp-list.md#dynamic-resizing`

File: docs/spec/amp-shadow-doc.md [hunk 2]
```diff
@@ -127,6 +127,6 @@ We tested with [WebComponents.js polyfill](https://github.com/webcomponents/webc
 
 ## Examples and references
 
-See [pwa.js](../examples/pwa/pwa.js) for examples of uses of boths APIs.
+See [pwa.js](../examples/pwa/pwa.js) for examples of uses of both APIs.
```
Comment (powerivq) [inline docs/spec/amp-shadow-doc.md:130]: ../../examples/pwa/pwa.js

File: docs/spec/amp-actions-and-events.md [hunk 3]
```diff
@@ -542,7 +542,7 @@ event.response</pre></td>
   </tr>
   <tr>
     <td><code>changeToLayoutContainer</code></td>
-    <td>Update's <code>amp-list</code>'s layout to <code>layout="CONTAINTER"</code> to allow <a href="../extensions/amp-list/amp-list.md#dynamic-resizing">dynamic resizing</a>.</td>
+    <td>Update's <code>amp-list</code>'s layout to <code>layout="CONTAINER"</code> to allow <a href="../extensions/amp-list/amp-list.md#dynamic-resizing">dynamic resizing</a>.</td>
```
Comment (powerivq) [inline docs/spec/amp-actions-and-events.md:545]: same

../../

**Review by powerivq — CHANGES_REQUESTED** [review]
LGTM, but the check is failing for the reasons given. Could you help pushing these 3 changes?

File: docs/spec/amp-email-actions-and-events.md (hunk 1 above)
Comment (szepeviktor) [reply docs/spec/amp-email-actions-and-events.md:434]: You make me work!
Nowadays I am looking work. I'd pay you €3/hour if you give me some.

Comment (szepeviktor) [reply docs/spec/amp-email-actions-and-events.md]: Done 🍏

File: docs/spec/amp-shadow-doc.md (hunk 2 above)
Comment (szepeviktor) [reply docs/spec/amp-shadow-doc.md]: Done 🍏

File: docs/spec/amp-actions-and-events.md (hunk 3 above)
Comment (szepeviktor) [reply docs/spec/amp-actions-and-events.md]: Done 🍏

File: docs/spec/amp-cache-transform.md [hunk 4]
```diff
@@ -196,13 +196,13 @@ publisher additional assurance that the JS is not an arbitrary payload. It would
 nice to get rid of this dependency; something like [signature-based
 SRI](https://github.com/mikewest/signature-based-sri) might be feasible.
 
-## Interaction with content negotation
+## Interaction with content negotiation
```
Comment (powerivq) [inline docs/spec/amp-cache-transform.md:199]: Line 178: substitute draft-ietf-httpbis-variants.html address with https://datatracker.ietf.org/doc/draft-ietf-httpbis-variants/00/

File: docs/spec/amp-html-format.md [hunk 5]
```diff
@@ -159,7 +159,7 @@ AMP HTML documents MUST
 -   <a name="canon"></a>contain a `<link rel="canonical" href="$SOME_URL">` tag inside their head that points to the regular HTML version of the AMP HTML document or to itself if no such HTML version exists. [🔗](#canon)
 -   <a name="chrs"></a>contain a `<meta charset="utf-8">` tag as the first child of their head tag. [🔗](#chrs)
 -   <a name="vprt"></a>contain a `<meta name="viewport" content="width=device-width">` tag inside their head tag. It's also recommended to include `minimum-scale=1` and `initial-scale=1`. [🔗](#vprt)
--   <a name="scrpt"></a>contain a `<script async src="https://cdn.ampproject.org/v0.js"></script>` tag inside their head tag. [🔗](#scrpt)
+-   <a name="script"></a>contain a `<script async src="https://cdn.ampproject.org/v0.js"></script>` tag inside their head tag. [🔗](#script)
```
Comment (powerivq) [inline docs/spec/amp-html-format.md:162]: Remove https://developer.mozilla.org/en-US/docs/Web/SVG/Element/solidcolor link from this file

**Review by powerivq — CHANGES_REQUESTED** [review]
Sorry but still got these errors to fix

File: docs/spec/amp-cache-transform.md [hunk 6]
```diff
@@ -33,7 +33,7 @@ subresources as well, they must also be served from a coordinating AMP cache, in
 order to preserve privacy. In order for those subresources to be _useful_, they
```
[the remark this answers is not in the document]
Comment (szepeviktor) [reply docs/spec/amp-cache-transform.md:32]: Done 🍏

File: docs/spec/amp-cache-transform.md (hunk 4 above)
Comment (szepeviktor) [reply docs/spec/amp-cache-transform.md:198]: Done 🍏

File: docs/spec/amp-html-format.md (hunk 5 above)
Comment (szepeviktor) [reply docs/spec/amp-html-format.md:162]: Done 🍏

File: docs/spec/amp-cache-transform.md [hunk 7]
```diff
@@ -88,7 +88,7 @@ For each identifier:
 
 1.  If the identifier contains a `v` parameter, then its value represents a set
     of AMP transform versions. The server should respond with an SXG only if it
-    can produce one of the versions in that set (see [Version negotation](#version-negotiation)).
+    can produce one of the versions in that set (see [Version negotiation](#version-negotiation)).
 2.  If the identifier contains any parameters besides those mentioned above,
     then this identifier cannot be satisfied. The server should attempt to match
     the next one. (This reserves the parameter space for future additional
```
[the remark this answers is not in the document]
Comment (szepeviktor) [reply docs/spec/amp-cache-transform.md:93]: Done 🍏

**Review by powerivq — APPROVED** [review]

File: docs/spec/amp-3p-video.md
Comment (orunoterry21-tech) [inline docs/spec/amp-3p-video.md]: [](url)

**Pull request merged**